jscomps/EmbedliteDownloadManager.js

```diff
@@ -15,6 +15,7 @@ const Cu = Components.utils;
 const Cr = Components.results;
 
 const { XPCOMUtils } = ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
+const { PrivateBrowsingUtils } = ChromeUtils.import("resource://gre/modules/PrivateBrowsingUtils.jsm");
```

Comment:
Not needed as download object itself carries this information. See https://github.com/sailfishos-mirror/gecko-dev/blob/600df6aa643726626d2eab36bc0f6e34bbe5ac2a/toolkit/components/downloads/Downloads.jsm#L80